tutorials/build_system/deployment.md

```diff
@@ -38,8 +39,20 @@ git update-index --assume-unchanged gradle.properties
 
 If your project already has a custom `gradle.properties` file, you may create a custom `*.properties` file, and load it manually. For example:
 
+```groovy
```

Comment:
I'd get rid of this completely. If you put your properties in the `gradle.properties` that will be automatically imported into `project` properties. See https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties

Reply:
Thanks for the review! 
Yes I know, but let me write down my original thought: In this case we are talking about username+password, which you do not want to commit to a VCS. But I think it is quite common that the `gradle.properties` file _is_ committed to a VCS, so you wouldn't want to add it to that file but keep it in a separate `credentials.properties` file, say, which you then don't commit to a VCS. Do you agree? If so, then this idea should maybe replace the "If your project already has a custom gradle.properties file" because I think that would not be a real reason for doing this, for the reason you mentioned.

Reply:
>  Do you agree?

Yes, it's a valid point and valid approach for sure.

I usually set empty username and password in `gradle.properties` and override them on the CI using system property `-Dorg.gradle.project.myPropertyName=username` or via environment variable ORG_GRADLE_PROJECT_myPropertyName.

Reply:
Oh, but that is much better than trying not to commit a certain file! I didn't know that was possible. I tested both ways, the argument and environment and they both work. If you don't mind I'm going to write that down and replace the mentioned paragraph, unless it's already written down elsewhere and I should link to it.

Reply:
In fact, doing it this way you don't need to put the properties in the `gradle.properties` file, so why use that at all?

Reply:
> In fact, doing it this way you don't need to put the properties in the gradle.properties file, so why use that at all?

To have default values. If you reference these properties in the build script and do not pass anything via environment or argument, you'll get a compilation error.

Yes, please, write it down too.

Reply:
Ah of course that is correct, forgot that sorry.
I think I have applied all suggestions now, I also resolved the merge conflict which was introduced because Yann noticed the same wrong releases url (ac39402cba623fb071c185051fb69821d219a43c).
Thanks again and no worries about any delay in reviewing.